src/utils/dates.js

```diff
@@ -11,9 +11,24 @@ export function getFutureDate(offset) {
 	return new Date(Date.now() + offset * ONE_DAY_IN_MILLISECONDS);
 }
 
-export function getDaysBetweenDates(previousPurchase, currentDate) {
-	if (!currentDate) {
-		currentDate = Date.now();
+export function getDaysBetweenDates(previousPurchase, futurePurchase) {
+	const currentDate = new Date();
+	// console.log(previousPurchase, futurePurchase);
+	if (previousPurchase && futurePurchase) {
+		//console.log(`${previousPurchase && futurePurchase} previous and current`)
+		return Math.round(
+			(futurePurchase.toDate() - previousPurchase.toDate()) /
+				ONE_DAY_IN_MILLISECONDS,
+		);
+	}
+	if (!previousPurchase) {
+		// console.log(previousPurchase < currentDate);
+		return Math.round(
+			(previousPurchase.toDate() - currentDate) / ONE_DAY_IN_MILLISECONDS,
```

Comment:
`if (!previousPurchase)` what would be the result of `previousPurchase.toDate() `?